content/en/docs/tasks/debug-application-cluster/resource-usage-monitoring.md

```diff
@@ -46,73 +47,61 @@ monitoring statistics by default:
+resource metrics api.  This api is served at `/metrics/resource/v1alpha1` on the 
+kubelet's authenticated and read-only ports.
 
 ### cAdvisor
 
-cAdvisor is an open source container resource usage and performance analysis agent. It is purpose-built for containers and supports Docker containers natively. In Kubernetes, cAdvisor is integrated into the Kubelet binary. cAdvisor auto-discovers all containers in the machine and collects CPU, memory, filesystem, and network usage statistics. cAdvisor also provides the overall machine usage by analyzing the 'root' container on the machine.
+cAdvisor is an open source container resource usage and performance analysis 
+agent. It is purpose-built for containers and supports Docker containers 
+natively. In Kubernetes, cAdvisor is integrated into the Kubelet binary. 
+cAdvisor auto-discovers all containers in the machine and collects CPU, memory, 
+filesystem, and network usage statistics. cAdvisor also provides the overall 
+machine usage by analyzing the 'root' container on the machine.
 
-Kubelet exposes a simple cAdvisor UI for containers on a machine, via the default port 4194.
-The picture below is an example showing the overall machine usage. However, this feature has been marked
-deprecated in v1.10 and completely removed in v1.12.
-
-![cAdvisor](/images/docs/cadvisor.png)
-
-Starting from v1.13, you can [deploy cAdvisor as a DaemonSet](https://github.com/google/cadvisor/tree/master/deploy/kubernetes) for an access to the cAdvisor UI.
+For details on accessing the cAdvisor UI, please consult the cAdvisor
+ [documentation](https://github.com/google/cadvisor). 
 
 ## Full metrics pipelines
 
-Many full metrics solutions exist for Kubernetes.
-
-### Prometheus
-
-[Prometheus](https://prometheus.io) can natively monitor kubernetes, nodes, and prometheus itself.
-The [Prometheus Operator](https://coreos.com/operators/prometheus/docs/latest/)
-simplifies Prometheus setup on Kubernetes, and allows you to serve the
-custom metrics API using the
-[Prometheus adapter](https://github.com/directxman12/k8s-prometheus-adapter).
-Prometheus provides a robust query language and a built-in dashboard for
-querying and visualizing your data. Prometheus is also a supported
-data source for [Grafana](https://prometheus.io/docs/visualization/grafana/).
-
-### Sysdig
-[Sysdig](http://sysdig.com) provides full spectrum container and platform intelligence, and is a
-true container native solution. Sysdig pulls together data from system calls, Kubernetes events,
-Prometheus metrics, statsD, JMX, and more into a single pane that gives you a comprehensive picture 
-of your environment. Sysdig also provides an API to query for providing robust and customizable 
-solutions. Sysdig is built on Open Source. [Sysdig and Sysdig Inspect](https://sysdig.com/opensource/inspect/) give you the 
-ability to freely perform troubleshooting, performance analyis and forensics. 
-
-### Google Cloud Monitoring
```

Comment:
Looks to me this subsection has been drastically trimmed which is fine. However, we may want to remove the linked image resource if we are doing so.

Reply:
@tenqm Yes - the image has been marked for deletion (it's the second file changed, way at the bottom of the Files changed tab)